Dockerfile.monitoring

```diff
@@ -23,7 +23,7 @@ ARG project_id
 RUN apt-get update && \
     apt-get install -y python3-pip vim curl 
 
-# RUN source activate rapids
+# RUN source activate merlin
```

Comment:
we've stopped using conda on the merlin containers as of 0.5.1 - we can probably remove this line (also this is based off the ```google/cloud-sdk``` container which won't have merlin?)

Reply:
Yes, removed , thanks for pointing this out.